python/pyspark/sql/tests.py

```diff
@@ -30,6 +30,19 @@
 import functools
 import time
 import datetime
+import traceback
+
+if sys.version_info[:2] <= (2, 6):
```

Comment:
I think it is a preference. I don't think either is particularly better. Just per documentation. it sounds `version_info` is preferred - https://docs.python.org/2/library/sys.html#sys.version

> Do not extract version information out of it, rather, use `version_info` and the functions 

although we are not "extract"ing anyway ...